torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1572,12 +1632,9 @@ PyTorchModelLoader::PyTorchModelLoader(
     const torch::jit::Value *inputValue = graphInputValues[i];
     const c10::IValue inputIValue = inputs.at(i);
     if (inputIValue.isTensor()) {
-      const auto ptTensorType = c10::TensorType::create(inputIValue.toTensor());
       glow::Placeholder *PH;
-      if (auto resOrErr = loadValue(inputValue, *ptTensorType)) {
-        PH = std::move(*resOrErr);
-      } else {
-        error = resOrErr.takeError();
+      if ((error = createPlaceholderFromTensor(inputValue, PH,
```

Comment:
RETURN_IF_ERR

Reply:
It's a constructor, return error is not allowed.